README.md

```diff
@@ -85,7 +85,7 @@ In [IBM Cloud](https://cloud.ibm.com):
 
 ## 3. Building and Running
 
-> Note: This has been compiled and tested using Unity 2018.3.0f2 and Watson Unity SDK version `v3.1.0`.
+> Note: This has been compiled and tested using Unity `2018.3.0f2` and Unity `2019.1.0f2` and Watson Unity SDK version `v3.1.0`.
```

Comment:
suggest making this a bulleted list - easier to read

Reply:
Well, I've never had >1 version before, and may never again. Not sure it's worth making a list.